spec/test_resuilt_parser_spec.rb

```diff
@@ -17,5 +17,16 @@
           failures: 1
       })
     end
+
+    it "properly parses the xcodebuild output" do
+      output = "<?xml version='1.0' encoding='UTF-8'?>
+        <testsuites tests='2' failures='1'/>"
+
+      result = Scan::TestResultParser.new.parse_result(output)
+      expect(result).to eq({
+          tests: 2,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.